docs/markdown/authoritative/settings.md

```diff
@@ -635,6 +635,15 @@ method to distribute the zone data to the slaves), then set `only-notify` to an
 value and specify the notification targets explicitly using [`also-notify`](#also-notify)
 and/or `ALSO-NOTIFY` domain metadata to avoid this potential bottleneck.
 
+## `disable-notification-warning`
```

Comment:
I don't like the double negative here (`disable-` and the default of 'no'), maybe rename this to 'log-notification-warning` and set it to 'yes' by default?